src/spec_tests/ssz_static/root.zig

```diff
@@ -26,42 +22,105 @@ fn loadFromFile(file_path: []const u8) !Yaml {
 const Roots = struct {
     root: [32]u8,
 };
+
+const structMultiPhase = union {
+    Attestation: types.Attestation,
+    BeaconBlockBody: types.BeaconBlockBody,
+    BeaconState: types.BeaconState,
+    BlobIdentifier: types.BlobIdentifier,
+    BlobSidecar: types.BlobSidecar,
+    BLSToExecutionChange: types.BLSToExecutionChange,
+    ConsolidationRequest: types.ConsolidationRequest,
+    ContributionAndProof: types.ContributionAndProof,
+    DepositRequest: types.DepositRequest,
+    ExecutionPayload: types.ExecutionPayload,
+    ExecutionPayloadHeader: types.ExecutionPayloadHeader,
+    HistoricalSummary: types.HistoricalSummary,
+    LightClientBootstrap: types.LightClientBootstrap,
+    LightClientFinalityUpdate: types.LightClientFinalityUpdate,
+    LightClientHeader: types.LightClientHeader,
+    LightClientOptimisticUpdate: types.LightClientOptimisticUpdate,
+    LightClientUpdate: types.LightClientUpdate,
+    PendingBalanceDeposit: types.PendingBalanceDeposit,
+    PendingConsolidation: types.PendingConsolidation,
+    PendingPartialWithdrawal: types.PendingPartialWithdrawal,
+    PowBlock: types.PowBlock,
+    SignedBLSToExecutionChange: types.SignedBLSToExecutionChange,
+    SignedContributionAndProof: types.SignedContributionAndProof,
+    SignedVoluntaryExit: types.SignedVoluntaryExit,
+    SyncAggregate: types.SyncAggregate,
+    SyncCommittee: types.SyncCommittee,
+    SyncCommitteeContribution: types.SyncCommitteeContribution,
+    SyncCommitteeMessage: types.SyncCommitteeMessage,
+    Withdrawal: types.Withdrawal,
+    WithdrawalRequest: types.WithdrawalRequest,
+};
+
+const altair = union {
+    SyncAggregatorSelectionData: types.SyncAggregatorSelectionData,
+};
+
 // test cases for all phases
-const CommonUnion = union {
+const commonUnion = union {
+    // AggregateAndProof: types.AggregateAndProof,
+    // AttestationData: types.AttestationData,
+    // AttesterSlashing: types.AttesterSlashing,
+    // BeaconBlock: types.BeaconBlock,
+    // BeaconBlockHeader: types.BeaconBlockHeader,
+    Checkpoint: types.Checkpoint,
+    // Deposit: types.Deposit,
+    // DepositData: types.DepositData,
+    // DepositMessage: types.DepositMessage,
+    // Eth1Block: types.Eth1Block,
+    // Eth1Data: types.Eth1Data,
     Fork: types.Fork,
+    ForkData: types.ForkData,
+    // HistoricalBatch: types.HistoricalBatch,
+    // IndexedAttestation: types.IndexedAttestation,
+    // PendingAttestation: types.PendingAttestation,
+    // ProposerSlashing: types.ProposerSlashing,
+    // SignedBeaconBlock: types.SignedBeaconBlock,
+    // SignedBeaconBlockHeader: types.SignedBeaconBlockHeader,
+    // SigningData: types.SigningData,
+    SyncAggregatorSelectionData: types.SyncAggregatorSelectionData, // only exist in altair or later
+    // Validator: types.Validator,
+    VoluntaryExit: types.VoluntaryExit,
 };
 
+const forks = [_][]const u8{ "phase0", "altair", "bellatrix", "capella", "deneb", "electra" };
```

Comment:
Can it be function in `ForkType` in consensus/types file

Reply:
this is the dir name of test